.circleci/config.yml

```diff
@@ -2,10 +2,10 @@ version: 2
 jobs:
   build:
     environment:
-      GRADLE_OPTS: -Dorg.gradle.jvmargs="-Xmx2048m -XX:+HeapDumpOnOutOfMemoryError"
+      GRADLE_OPTS: -Dorg.gradle.jvmargs="-Xmx2048m -XX:+HeapDumpOnOutOfMemoryError" -Dkotlin.compiler.execution.strategy=in-process
       _JAVA_OPTIONS: -Xms512m -Xmx1024m
     docker:
-      - image: circleci/android:api-28-alpha
+      - image: circleci/android:api-29
```

Comment:
why version 29, which is different than the target version specified in build.gradle? API 29 requires Java 9, correct?

Reply:
AFAIK circleci doesn't have an image with version 30 of the Android SDK published yet. Java 9 installation is required to run Robolectric tests that target 29 and above. Not sure about API 29 itself enforcing any JDK requirements.